File: src/yanzm/example/adaptersamples/ArrayAdapterSample3.java
package yanzm.example.adaptersamples;

import android.app.ListActivity;
import android.os.Bundle;
import android.widget.ArrayAdapter;

public class ArrayAdapterSample3 extends ListActivity {

	public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        String[] data = {"夏目漱石", "太宰治"};
        
        ArrayAdapter<String> arrayAdapter = new ArrayAdapter<String>(this, R.layout.list_item3, data);
        setListAdapter(arrayAdapter);
    }
}

File: app/src/main/java/yanzm/example/adaptersamples/ArrayAdapterSample3.kt
package yanzm.example.adaptersamples

import android.os.Bundle
import android.widget.ArrayAdapter
import android.widget.ListView
import androidx.appcompat.app.AppCompatActivity

class ArrayAdapterSample3 : AppCompatActivity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val listView = ListView(this)
        setContentView(listView)

        val data = arrayOf("夏目漱石", "太宰治")
        listView.adapter = ArrayAdapter(this, R.layout.list_item3, data)
    }
}
